src/core.c/RakuAST/Deparse.rakumod

```diff
@@ -1007,7 +1007,7 @@ CODE
     }
 
     multi method deparse(RakuAST::MetaInfix::Negate:D $ast --> Str:D) {
-        self.syn-infix(self.deparse($ast.infix) ~ '!')
+        self.syn-infix('!' ~ self.deparse($ast.infix))
```

Comment:
To make localization work correctly, the '!' should be *before* the `self.syn-infix`

Reply:
```
[0] > '1 !(cont) 2'.AST.DEPARSE: <DE>
1 !(enthält) 2
```
Seems like it works anyway due to `self.deparse($ast.infix)` already handling the localization ( which also seems to be the case in the methods below )
Though since `'!'` isn't localized I suppose it could be `self.hsyn('infix', '!' ~ self.deparse($ast.infix))`
Or rather `self.hsyn('infix', '!' ~ self.xsyn('infix', self.deparse($ast.infix)))` like in the method right above ( But I feel like the call to `xsyn` there isn't really needed? )

In short, isn't `deparse(RakuAST::Infix:D $ast --> Str:D)` (line 963) already calling `self.syn-infix` (and in turn `xsyn`) for the part that should be localized? And if it isn't, isn't it also broken in the methods below?
( Apologies if this is nonsense, I'm trying to understand )

Reply:
No apologies needed.  I'm very grateful for these extra pair of eyes on this code, as there seem to be a number of thinkos embedded here.

Investigating...